doc/proxy-clients.rst

```diff
@@ -97,12 +97,28 @@ You can also pass some options to the Varnish client:
 * ``tags_header`` (X-Cache-Tags): Allows you to change the HTTP header used for
   tagging. If you change this, make sure to use the correct header name in your
   :doc:`proxy server configuration <proxy-configuration>`;
+* ``tags_purge_header`` (key): Allows you to change the HTTP header used for
+  purging tags when ``tag_mode`` is set to one of the ``purge*`` options. If
+  you change this, make sure to use the correct header name in your
+  :doc:`proxy server configuration <proxy-configuration>`;
+* ``tag_mode`` (ban): Makes it possible to invalidate tags
+  using purge when for instance using ``xkey`` instead of ban. Supports three
+  modes: ``ban``, ``purge`` and ``purge_single`` for purge per tag.
```

Comment:
can we rephrase this? "``tag_mode`` (ban): Specify the method how tags are invalidated. ``ban`` uses the built-in BAN functionality of Varnish, ``purge`` uses the xkey Varnish module, and ``purge_single`` ???" (not sure what the difference between purge and purge_single is, we should say this here (no long explanation, but a short note))